NLog.Etw/RegisterEvents/ReadMe.txt

```diff
@@ -0,0 +1,3 @@
+﻿RegisteredEvents.EXE is a seld-extracting, self-installing package used to register the NLog ETW provider and channel-based log files.
```

Comment:
do we need this files: readme.txt, RegisterEvents.EXE,  /RegisterEvents.SED, _EventRegisterUsersGuide.docx .bat files etc